rospy_tutorials/test/test-talker-listener-with-timer.launch

```diff
@@ -1,4 +1,9 @@
 <launch>
+
   <node name="talker" pkg="rospy_tutorials" type="talker_timer.py" />
-  <test test-name="talker_listener_test" pkg="rospy_tutorials" type="talker_listener_test.py" />
+
+  <test test-name="talker_listener_test_with_timer"
+        name="talker_listener_test_with_timer"
```

Comment:
I don't know - you might want to try it.

But this test doesn't have anything like that.